modules/labs-react/common/lib/Box.tsx

```diff
@@ -22,7 +22,7 @@ export type BoxProps = BorderStyleProps &
     children?: React.ReactNode;
   };
 
-const omittedProps = ['display', 'color'];
+const omittedProps = ['display', 'color', 'height', 'overflow', 'width'];
```

Comment:
I saw these oddly being passed through with the `SidePanel` section using `Box`.